### ISSUE TITLE
pretrained weight

### ISSUE THREAD

**Issue by mos3ab8** [long123524/BsiNet-torch#8]
Could you please upload the pretrained weight to google drive
Thank you

**Comment by long123524** [comment]
Thanks for your concern. We have done.

**Comment by long123524** [comment]
> Could you please upload the pretrained weight to google drive Thank you

Thanks for your concern. We have done.

**Comment by mos3ab8** [comment]
Thank you for sharing.
Could you also upload the GF2 image (1m) to google drive.

Additionally, I would appreciate it if you could provide instructions on how to apply the model to a publicly available dataset, such as the one accessible at this link: https://easy.dans.knaw.nl/ui/datasets/id/easy-dataset:200357/tab/2. Thank you very much.

**Comment by long123524** [comment]
> Thank you for sharing. Could you also upload the GF2 image (1m) to google drive.
> 
> Additionally, I would appreciate it if you could provide instructions on how to apply the model to a publicly available dataset, such as the one accessible at this link: https://easy.dans.knaw.nl/ui/datasets/id/easy-dataset:200357/tab/2. Thank you very much.

Thanks again for your concern. We have uploaded the GF2 image to Google Drive.  Applying our method to a public dataset is not difficult, you first need to pre-process the dataset to obtain the image and label pair, and then train our model. The proposed BsiNet obtained satisfactory results on the medium-resolution images (e.g., Sentinel-2), you can find more detailed information in the supplementary document of our paper.
